src/app/common/stores/applications/SsvWeb/Cluster.store.ts

```diff
@@ -55,10 +56,16 @@ class ClusterStore extends BaseStore {
 
   getClusterNewBurnRate(cluster: any, newAmountOfValidators: number) {
     const ssvStore: SsvStore = this.getStore('SSV');
-    const operatorsFeePerYear = cluster.operators.reduce((acc: number, operator: IOperator) => Number(acc) + Number(formatNumberToUi(ssvStore.getFeeForYear(fromWei(operator.fee)))), [0]);
+    const operatorStore: OperatorStore = this.getStore('Operator');
+    const operatorsFeePerYear = Object.values(operatorStore.selectedOperators).reduce((acc: number, operator: IOperator) => Number(acc) + Number(formatNumberToUi(ssvStore.getFeeForYear(fromWei(operator.fee)))), 0);
```

Comment:
Why do you need to call formalNumberToUi?

Reply:
V